src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/HtmlCSharp/DefaultLSPProgressListenerTest.cs

```diff
@@ -227,10 +229,11 @@ public async Task TryListenForProgress_MultipleProgressNotificationReported()
 
             // Assert
             Assert.True(listenerAdded);
-            receivedResults.Sort();
+            var sortedResults = receivedResults.ToList();
```

Comment:
Maybe combine this line and the next one?

Reply:
I don't think that would be possible given `Sort()` [returns void](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.sort?view=netcore-3.1), sorting the list in place.